### PR TITLE
ENGDOCS-1401

### PR DESCRIPTION
**Pull request by aevesdocker**
Splits out the content that was on two tabs to two separate pages

### CODE REVIEW

File: desktop/hardened-desktop/enhanced-container-isolation/faq.md [hunk 1]
```diff
@@ -1,79 +1,70 @@
...
-<div id="tab3" class="tab-pane fade in active" markdown="1">
+### Do I need to change the way I use Docker when Enhanced Container Isolation is switched on?
 
-#### Do I need to change the way I use Docker when Enhanced Container Isolation is enabled?
+No, you can continue to use Docker as usual. 
 
-No, you can continue to use Docker as usual. Enhanced Container Isolation will be mostly transparent to you.
-
-#### Do all container workloads work well with Enhanced Container Isolation?
+### Do all container workloads work well with Enhanced Container Isolation?
```
Comment (ChrisChinchilla) [inline desktop/hardened-desktop/enhanced-container-isolation/faq.md:12]: @aevesdocker Actually, as you're changing them anyway, technically these should all be H2s

File: desktop/hardened-desktop/enhanced-container-isolation/limitations.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+---
+title: Limitations
+description: Limitations of Enhanced Container Isolation
+keywords: enhanced container isolation, security, sysbox, known issues, Docker Desktop
+toc_max: 2
+---
+
+### ECI support for WSL
```
Comment (ChrisChinchilla) [inline desktop/hardened-desktop/enhanced-container-isolation/limitations.md:8]: This should also be H2

Comment (aevesdocker) [reply desktop/hardened-desktop/enhanced-container-isolation/limitations.md:8]: Same comment as above

File: desktop/hardened-desktop/enhanced-container-isolation/limitations.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+| Docker Desktop Linux VM protected from user access | No           | Yes              | On WSL, users can access Docker Engine directly or bypass Docker Desktop security settings. |
+| Docker Desktop Linux VM has a dedicated kernel     | No           | Yes              | On WSL, Docker Desktop can't guarantee the integrity of kernel level configs. |
+
+In general, using ECI with Hyper-V is more secure than with WSL 2. But WSL 2
+offers advantages for performance and resource utilization on the host machine,
+and it's an excellent way for users to run their favorite Linux distro on
+Windows hosts and access Docker from within (see Docker Desktop's WSL distro
+integration feature, enabled via the Dashboard's **Settings** > **Resources** > **WSL Integration**).
+
+### Docker build and buildx has some restrictions
```
Comment (ChrisChinchilla) [inline desktop/hardened-desktop/enhanced-container-isolation/limitations.md:59]: I sort of feel like this should be "have", but I might be wrong

**Review by ChrisChinchilla — COMMENTED** [review]
I have comments, they're up to you though

File: desktop/hardened-desktop/enhanced-container-isolation/limitations.md (hunk 3 above)
Comment (aevesdocker) [reply desktop/hardened-desktop/enhanced-container-isolation/limitations.md:59]: yes good spot. There should also be some capitals...

**Comment by aevesdocker** [comment]
Thanks @ChrisChinchilla! Made some of the changes - ready for final review

**Review by ChrisChinchilla — APPROVED** [review]

**Pull request merged**